pkg/virtctl/create/vm/vm_test.go

```diff
@@ -1352,7 +1441,7 @@ chpasswd: { expire: False }`
 
 		It("Volume specified in InferInstancetypeFromFlag should exist", func() {
 			out, err := runCmd(setFlag(InferInstancetypeFromFlag, "does-not-exist"))
-			Expect(err).To(MatchError("there is no volume with name 'does-not-exist'"))
+			Expect(err).To(MatchError("there is no volume with name \"does-not-exist\""))
```

Comment:
define this error as constant?

Reply:
Pushed another commit that introduces consts for values used multiple times.